tiapp.xml

```diff
@@ -59,7 +59,7 @@
                 </activity>
             </application>
             <supports-screens android:anyDensity="true" android:largeScreens="true" android:normalScreens="true" android:smallScreens="true" xlargeScreens="true"/>
-            <uses-sdk android:installLocation="preferExternal" android:minSdkVersion="29" android:targetSdkVersion="34"/>
+            <uses-sdk android:installLocation="preferExternal" android:minSdkVersion="30" android:targetSdkVersion="35"/>
```

Comment:
careful with target 35. Please check it on a device to see if the edge-to-edge display looks fine. Otherwise: https://github.com/tidev/titanium-sdk/issues/14111 and add the `windowOptOutEdgeToEdgeEnforcement` part to your theme

Reply:
Great catch @m1ga . Should I continue targeting 34 for now?

Reply:
I would remove `android:targetSdkVersion=""` and let the SDK do that for you (34 is the default anyways). Keep the minSDK. 
There is already a PR for target 35 with `windowOptOutEdgeToEdgeEnforcement` so it will look like the current Ti 12.7.0. Hopefully that gets merged soon and then you can update it when you do an update after November

Reply:
Great, thanks for this 👍